tests/test_url_conversions.py

```diff
@@ -51,6 +55,14 @@ def test_roundtrip_rel(self):
         # should have reconstructed it by this point
         self.assertEqual(os.path.normpath(result), MEDIA_EXAMPLE_PATH_REL)
 
+    def test_windows_urls(self):
+        for url in (ENCODED_WINDOWS_URL, WINDOWS_URL):
+            print(f"Original URL: {url}")
+            self.assertTrue(url.startswith("file://"))
+            processed_url = otio.url_utils.filepath_from_url(url)
+            print(f"Processed URL Path: {processed_url}")
```

Comment:
I'll remove the prints and I added `self.assertTrue(url.startswith("file://"))` because the other unit tests were doing something similar. I can remove it as well though.